loader/api.js

```diff
@@ -70,12 +71,16 @@ mapOwn('actionText,setName,setAttribute,save,ignore,onEnd,getContext,end,get'.sp
 
 function apiCall (name, notSpa, bufferGroup) {
   return function () {
+    var cleanedName = name.replace('api-', '')
+    console.log('an API was called, increment counter for', cleanedName)
+    customMetrics.incrementCounter('API/' + cleanedName + '/called', true)
```

Comment:
It might be a little better to have a convenience method for recording supportability metrics, something like what the Node agent has with the [`recordSupportability` function](https://github.com/newrelic/node-newrelic/blob/68da8b2343e5a349f1718ab09c675fd2ab08737e/lib/agent.js#L842)